pycaching/trackable.py

```diff
@@ -179,6 +199,7 @@ def load(self):
         self.owner = root.find(id="ctl00_ContentBody_BugDetails_BugOwner").text
         self.goal = root.find(id="TrackableGoal").text
         self.description = root.find(id="TrackableDetails").text
+        self.kmlurl = root.find(id="ctl00_ContentBody_lnkGoogleKML").get("href")
```

Comment:
Can you please make 2 more changes?

1. rename the `kmlurl` to `kml_url` according to [PEP8](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables):

   > ### Method Names and Instance Variables
   > Use the function naming rules: lowercase with words separated by underscores as necessary to improve readability.

1. Make the variable "non-public" (as defined in PEP8) – add one leading underscore – so the final name will be `_kml_url`. Then you can delete the new `@property` and `@kmlurl.setter` methods and handle the potentially missing URL in the `get_KML()` method the same way like [in the `_load_log_page()` method](https://github.com/tomasbedrich/pycaching/blob/master/pycaching/trackable.py#L198-L199).

  The goal of this change is to hide this variable from `Trackable` public API.